lib/tests/lib.rs

```diff
@@ -0,0 +1,77 @@
+// Copyright 2023 Polyphene.
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use fvm_shared::error::ExitCode;
+use kythera_test_actors::wasm_bin::BASIC_TEST_ACTOR_BINARY;
+
+use kythera_common::abi::{Method, ABI};
+use kythera_lib::{Tester, WasmActor};
+
+const TARGET_WAT: &str = r#"
+;; Mock invoke function
+(module
+  (func (export "invoke") (param $x i32) (result i32)
+    (i32.const 1)
+  )
+)
+"#;
+
+#[test]
+fn test_tester() {
```

Comment:
wdyt of separating this test into two, one that instantiates and another that tests, so that we have better granularity? 
Rational is that instantiating the tester loads the builtin actors and creates an account for them.

Reply:
Yep okay ! Will do so !